pkg/scaffold/v2/main.go

```diff
@@ -27,9 +27,9 @@ import (
 )
 
 const (
-	apiPkgImportScaffoldMarker    = "// +kubebuilder:scaffold:imports"
-	apiSchemeScaffoldMarker       = "// +kubebuilder:scaffold:scheme"
-	reconcilerSetupScaffoldMarker = "// +kubebuilder:scaffold:builder"
+	ApiPkgImportScaffoldMarker    = "// +kubebuilder:scaffold:imports"
+	ApiSchemeScaffoldMarker       = "// +kubebuilder:scaffold:scheme"
+	ReconcilerSetupScaffoldMarker = "// +kubebuilder:scaffold:builder"
```

Comment:
Why are you exporting these constants?

Reply:
Because they are used out of here. See that in order to keep the project standard the controller and its tests suit were moved for the /pkg/scaffold/v2/controller dir.

Closing as sorted out.

Reply:
Where are they used?